go.mod

```diff
@@ -27,7 +27,7 @@ require (
 	github.com/rickb777/date v1.12.5-0.20200422084442-6300e543c4d9
 	github.com/sergi/go-diff v1.2.0
 	github.com/tombuildsstuff/giovanni v0.20.0
-	github.com/tombuildsstuff/kermit v0.20230530.1150329
+	github.com/tombuildsstuff/kermit v0.20230703.1101016
```

Comment:
@KenSpur since we're likely to update the Kermit SDK dependency outside of this PR (for other functionality), rather than including the SDK upgrade in this PR, would you mind splitting this into two PR's to first update the Kermit SDK dependency, and then add the new resources? This'd mean that there shouldn't be any merge conflicts in the `go.mod` / `go.sum` etc whilst developing these newer resources

Reply:
will do!

Reply:
@tombuildsstuff  pr is open #22390

Ill update this one tomorrow, when i continue my work

Reply:
@tombuildsstuff maybe we should use this pr to bump the version and add 1 data resource (so there is a dependecy on the kermit/sdk/iotcentral/2022-10-31-preview/iotcentral sdk)?